src/Http/Controllers/FilepondController.php

```diff
@@ -31,6 +31,12 @@ public function upload(Request $request)
     {
         $input = $request->file(config('filepond.input_name'));
         $file = is_array($input) ? $input[0] : $input;
+        
+        if($input == null){
```

Comment:
Looks good, could you just use a type safe check?
```suggestion
        if($input === null){
```

Reply:
type safe check as @Sopamo  suggested

Reply:
thanks for suggestion :)